demo/shaders/glsl/bloom_combine.frag

```diff
@@ -19,15 +19,109 @@ layout (set = 0, binding = 1) uniform texture2D in_blur;
 // ])]
 layout (set = 0, binding = 2) uniform sampler smp;
 
+// @[export]
+// @[internal_buffer]
+layout (set = 0, binding = 3) uniform Config {
+    int tonemapper_type;
+} config;
+
 layout (location = 0) in vec2 inUV;
 
 layout (location = 0) out vec4 out_sdr;
 
+// The code for ACESFitted was originally written by Stephen Hill (@self_shadow), who deserves all
```

Comment:
Since this is a non-trivial amount of code, could we separate it to a separate .glsl? Also I think we should add a link to the original material